src/main/scala/viper/server/core/DefaultVerificationExecutionContext.scala

```diff
@@ -0,0 +1,41 @@
+package viper.server.core
+
+import java.util.concurrent.{ExecutorService, Executors, Future, ThreadFactory}
+
+import akka.actor.{ActorSystem, BootstrapSetup}
+import akka.actor.setup.ActorSystemSetup
+
+import scala.concurrent.{ExecutionContext, ExecutionContextExecutorService}
+
+object DefaultVerificationExecutionContext {
+  // at least 2 threads are needed if the actor system is also using the same thread pool (as opposed to creating a
+  // actor system with the default executor like `ActorSystem(actorSystemName)`)
+  val minNumberOfThreads: Int = 2
```

Comment:
Could it be that this constant is different if the server creates instances of Carbon and Silicon? Silicon uses some concurrency so it may affect the thread pool, whereas Carbon, as far as I remember, does not.

Reply:
The execution context is not passed to the backend (not to increase the range of my changes even further). Hence, Silicon uses it's own thread pool and I think this constant is thus independent of the chosen backend